dace/sdfg/sdfg.py

```diff
@@ -298,6 +304,19 @@ class SDFG(OrderedDiGraph[SDFGState, InterstateEdge]):
 
     debuginfo = DebugInfoProperty(allow_none=True)
 
+    _pgrids = Property(dtype=dict,
```

Comment:
why does this have an underscore here

Reply:
It is supposed to be similar to `SDFG::_arrays`. There is also the corresponding `SDFG::pgrids` method.

Reply:
`_arrays` is a broken property too. It only has an underscore to avoid it showing up in the viewer, but that's not true anymore and it shows up anyway.